### ISSUE TITLE
Cannot install Simba with TF due to dependency conflicts

### ISSUE THREAD

**Issue by quirinkrabi** [sgoldenlab/simba#134]
Dear developers,

I am eager to use Simba, which looks highly promising for my research (social interaction between rats). Thank you for your efforts!

I want to use Simba with Tensorflow and DeepLabCut, as I already successfully worked with TF and DeepLabCut. I assume that this means I should install the "simba-uw-tf" version. By the way, I am not good at programming and command line usage, so I depend on using the GUI. 

I ran into problems installing it in a fresh Anaconda3 environment with Python 3.6, after "pip install simba-uw-tf".
It aborts with a dependency conflict error, see below.

I also tried "pip install simba-uw-tf=1.3.11", same error.

Could you please help me?

> By the way, I also tried installing from git repository directly, by doing the following steps in a fresh Anaconda environment:
> 	pip install wxpython
> 	conda install git
> 	pip install git+https://github.com/sgoldenlab/simba.git
> 	pip uninstall shapely
> 	conda install -c conda-forge shapely
> After that, I could not start simba by the command "simba".

Best wishes,
Quirin

**To Reproduce**
1. Generate fresh Anaconda3 environment under Windows 10
2. pip install simba-uw-tf or pip install simba-uw-tf=1.3.11

**Expected behavior**
I would expect the installation to work.

**Screenshots**
n/a

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Python Version 3.6. (Anaconda env selecting Python 3.6)
 - Are you using anaconda? Yes.
 

**Additional context**

### After **pip install simba-uw-tf**:

```
(Simba) C:\Users\************>pip install simba-uw-tf
Collecting simba-uw-tf
  Using cached Simba_UW_tf-1.3.12-py3-none-any.whl (7.3 MB)
Collecting yellowbrick==0.9.1
  Using cached yellowbrick-0.9.1-py2.py3-none-any.whl (282 kB)
Collecting numexpr==2.6.9
  Using cached numexpr-2.6.9-cp36-none-win_amd64.whl (91 kB)
Collecting statsmodels==0.9.0
  Using cached statsmodels-0.9.0-cp36-cp36m-win_amd64.whl (7.0 MB)
Collecting imgaug==0.4.0
  Using cached imgaug-0.4.0-py2.py3-none-any.whl (948 kB)
Collecting tqdm==4.30.0
  Using cached tqdm-4.30.0-py2.py3-none-any.whl (47 kB)
Collecting tabulate==0.8.3
  Using cached tabulate-0.8.3-py3-none-any.whl
Collecting dash==1.14.0
  Using cached dash-1.14.0-py3-none-any.whl
Collecting deepposekit==0.3.5
  Using cached deepposekit-0.3.5-py3-none-any.whl
Collecting pyyaml==5.3.1
  Using cached PyYAML-5.3.1-cp36-cp36m-win_amd64.whl (215 kB)
Collecting dash-core-components==1.10.2
  Using cached dash_core_components-1.10.2-py3-none-any.whl
Collecting dtreeviz==0.8.1
  Using cached dtreeviz-0.8.1-py3-none-any.whl
Collecting seaborn==0.9.0
  Using cached seaborn-0.9.0-py3-none-any.whl (208 kB)
Collecting dash-html-components==1.0.3
  Using cached dash_html_components-1.0.3-py3-none-any.whl
Collecting scikit-image==0.14.2
  Using cached scikit_image-0.14.2-cp36-none-win_amd64.whl (24.6 MB)
Collecting tensorflow-gpu==1.14.0
  Using cached tensorflow_gpu-1.14.0-cp36-cp36m-win_amd64.whl (287.7 MB)
Collecting pandas==0.25.3
  Using cached pandas-0.25.3-cp36-cp36m-win_amd64.whl (9.0 MB)
Collecting scipy==1.1.0
  Using cached scipy-1.1.0-cp36-none-win_amd64.whl (31.1 MB)
Collecting pyarrow==0.17.1
  Using cached pyarrow-0.17.1-cp36-cp36m-win_amd64.whl (21.6 MB)
Collecting imblearn==0.0
  Using cached imblearn-0.0-py2.py3-none-any.whl (1.9 kB)
Collecting cefpython3==66.0
  Using cached cefpython3-66.0-py2.py3-none-win_amd64.whl (68.2 MB)
Collecting plotly==4.9.0
  Using cached plotly-4.9.0-py2.py3-none-any.whl (12.9 MB)
Collecting opencv-python==3.4.5.20
  Using cached opencv_python-3.4.5.20-cp36-cp36m-win_amd64.whl (38.3 MB)
Collecting xgboost==0.90
  Using cached xgboost-0.90-py2.py3-none-win_amd64.whl (18.3 MB)
Collecting eli5==0.10.1
  Using cached eli5-0.10.1-py2.py3-none-any.whl (105 kB)
Collecting wxpython==4.0.4
  Using cached wxPython-4.0.4-cp36-cp36m-win_amd64.whl (23.0 MB)
Collecting dash-colorscales==0.0.4
  Using cached dash_colorscales-0.0.4-py3-none-any.whl
Collecting shap==0.35.0
  Using cached shap-0.35.0-cp36-cp36m-win_amd64.whl (309 kB)
Collecting deeplabcut==2.0.8
  Using cached deeplabcut-2.0.8-py3-none-any.whl (178 kB)
Collecting imutils==0.5.2
  Using cached imutils-0.5.2-py3-none-any.whl
Collecting numba==0.48.0
  Using cached numba-0.48.0-cp36-cp36m-win_amd64.whl (2.1 MB)
Collecting scikit-learn==0.22.2
  Using cached scikit_learn-0.22.2-cp36-cp36m-win_amd64.whl (6.5 MB)
Collecting graphviz==0.11
  Using cached graphviz-0.11-py2.py3-none-any.whl (17 kB)
Collecting matplotlib==3.0.3
  Using cached matplotlib-3.0.3-cp36-cp36m-win_amd64.whl (9.1 MB)
Collecting shapely==1.7
  Using cached Shapely-1.7.0-cp36-cp36m-win_amd64.whl (1.0 MB)
Collecting dash-color-picker==0.0.1
  Using cached dash_color_picker-0.0.1-py3-none-any.whl
Collecting h5py==2.9.0
  Using cached h5py-2.9.0-cp36-cp36m-win_amd64.whl (2.4 MB)
Collecting protobuf==3.6.0
  Using cached protobuf-3.6.0-cp36-cp36m-win_amd64.whl (1.1 MB)
Collecting Pillow==5.4.1
  Using cached Pillow-5.4.1-cp36-cp36m-win_amd64.whl (1.9 MB)
Collecting flask-compress
  Using cached Flask_Compress-1.10.1-py3-none-any.whl (7.9 kB)
Collecting dash-table==4.9.0
  Using cached dash_table-4.9.0-py3-none-any.whl
Collecting dash-renderer==1.6.0
  Using cached dash_renderer-1.6.0-py3-none-any.whl
Collecting Flask>=1.0.2
  Using cached Flask-2.0.1-py3-none-any.whl (94 kB)
Collecting future
  Using cached future-0.18.2-py3-none-any.whl
Collecting moviepy~=0.2.3.5
  Using cached moviepy-0.2.3.5.tar.gz (372 kB)
Collecting requests
  Using cached requests-2.26.0-py2.py3-none-any.whl (62 kB)
INFO: pip is looking at multiple versions of dash-table to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of dash-renderer to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of dash-html-components to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of dash-core-components to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of dash-colorscales to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of dash-color-picker to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of <Python from Requires-Python> to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of dash to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of cefpython3 to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of simba-uw-tf to determine which version is compatible with other requirements. This could take a while.
Collecting simba-uw-tf
  Using cached Simba_UW_tf-1.3.11-py3-none-any.whl (7.3 MB)
  Using cached Simba_UW_tf-1.3.10-py3-none-any.whl (7.3 MB)
  Using cached Simba_UW_tf-1.2.31-py3-none-any.whl (4.6 MB)
  Using cached Simba_UW_tf-1.2.30-py3-none-any.whl (4.6 MB)
  Using cached Simba_UW_tf-1.2.29-py3-none-any.whl (3.4 MB)
  Using cached Simba_UW_tf-1.2.28-py3-none-any.whl (3.4 MB)
  Using cached Simba_UW_tf-1.2.27-py3-none-any.whl (3.4 MB)
  Using cached Simba_UW_tf-1.2.26-py3-none-any.whl (3.4 MB)
  Using cached Simba_UW_tf-1.2.24-py3-none-any.whl (3.3 MB)
  Using cached Simba_UW_tf-1.2.23-py3-none-any.whl (3.3 MB)
  Using cached Simba_UW_tf-1.2.22-py3-none-any.whl (3.3 MB)
  Using cached Simba_UW_tf-1.2.21-py3-none-any.whl (3.3 MB)
  Using cached Simba_UW_tf-1.2.20-py3-none-any.whl (3.3 MB)
  Using cached Simba_UW_tf-1.2.19-py3-none-any.whl (3.3 MB)
  Using cached Simba_UW_tf-1.2.18-py3-none-any.whl (3.3 MB)
  Using cached Simba_UW_tf-1.2.17-py3-none-any.whl (3.3 MB)
  Using cached Simba_UW_tf-1.2.16-py3-none-any.whl (3.3 MB)
  Using cached Simba_UW_tf-1.2.15-py3-none-any.whl (3.3 MB)
  Using cached Simba_UW_tf-1.2.14-py3-none-any.whl (3.3 MB)
  Using cached Simba_UW_tf-1.2.13-py3-none-any.whl (3.3 MB)
  Using cached Simba_UW_tf-1.2.12.1-py3-none-any.whl (4.6 MB)
  Using cached Simba_UW_tf-1.2.12-py3-none-any.whl (3.3 MB)
  Using cached Simba_UW_tf-1.2.11-py3-none-any.whl (3.3 MB)
  Using cached Simba_UW_tf-1.2.10-py3-none-any.whl (3.3 MB)
  Using cached Simba_UW_tf-1.2.9.2-py3-none-any.whl (4.6 MB)
  Using cached Simba_UW_tf-1.2.9-py3-none-any.whl (3.3 MB)
  Using cached Simba_UW_tf-1.2.8-py3-none-any.whl (3.3 MB)
  Using cached Simba_UW_tf-1.2.7-py3-none-any.whl (3.3 MB)
  Using cached Simba_UW_tf-1.2.6-py3-none-any.whl (3.3 MB)
  Using cached Simba_UW_tf-1.2.5-py3-none-any.whl (3.3 MB)
  Using cached Simba_UW_tf-1.2.4.3-py3-none-any.whl (3.3 MB)
  Using cached Simba_UW_tf-1.2.4.2-py3-none-any.whl (3.3 MB)
  Using cached Simba_UW_tf-1.2.4.1-py3-none-any.whl (3.3 MB)
  Using cached Simba_UW_tf-1.2.3.13-py3-none-any.whl (3.3 MB)
  Using cached Simba_UW_tf-1.2.3.12-py3-none-any.whl (3.3 MB)
  Using cached Simba_UW_tf-1.2.3.11-py3-none-any.whl (3.3 MB)
  Using cached Simba_UW_tf-1.2.3.10-py3-none-any.whl (3.3 MB)
  Using cached Simba_UW_tf-1.2.3.9-py3-none-any.whl (3.3 MB)
  Using cached Simba_UW_tf-1.2.3.8-py3-none-any.whl (3.3 MB)
  Using cached Simba_UW_tf-1.2.3.7-py3-none-any.whl (3.3 MB)
  Using cached Simba_UW_tf-1.2.3.6-py3-none-any.whl (3.3 MB)
  Using cached Simba_UW_tf-1.2.3.5-py3-none-any.whl (3.3 MB)
  Using cached Simba_UW_tf-1.2.3.4-py3-none-any.whl (2.2 MB)
  Using cached Simba_UW_tf-1.2.3.3-py3-none-any.whl (2.2 MB)
  Using cached Simba_UW_tf-1.2.3-py3-none-any.whl (2.2 MB)
  Using cached Simba_UW_tf-1.2.2-py3-none-any.whl (2.2 MB)
  Using cached Simba_UW_tf-1.2.1-py3-none-any.whl (2.2 MB)
  Using cached Simba_UW_tf-1.2.0-py3-none-any.whl (1.9 MB)
  Using cached Simba_UW_tf-1.1.7-py3-none-any.whl (2.3 MB)
  Using cached Simba_UW_tf-1.1.6-py3-none-any.whl (2.3 MB)
  Using cached Simba_UW_tf-1.1.1-py3-none-any.whl (2.3 MB)
  Using cached Simba_UW_tf-1.1.0-py3-none-any.whl (2.3 MB)
  Using cached Simba_UW_tf-1.0.9-py3-none-any.whl (2.3 MB)
  Using cached Simba_UW_tf-1.0.7-py3-none-any.whl (2.3 MB)
  Using cached Simba_UW_tf-1.0.6-py3-none-any.whl (2.3 MB)
  Using cached Simba_UW_tf-1.0.5-py3-none-any.whl (2.3 MB)
  Using cached Simba_UW_tf-1.0.4-py3-none-any.whl (2.3 MB)
  Using cached Simba_UW_tf-1.0.3-py3-none-any.whl (2.3 MB)
Collecting deeplabcut==2.0.9
  Using cached deeplabcut-2.0.9-py3-none-any.whl (187 kB)
Collecting patsy
  Using cached patsy-0.5.1-py2.py3-none-any.whl (231 kB)
Collecting tables
  Using cached tables-3.6.1-2-cp36-cp36m-win_amd64.whl (3.2 MB)
Collecting imageio~=2.3.0
  Using cached imageio-2.3.0-py2.py3-none-any.whl (3.3 MB)
Collecting ruamel.yaml~=0.15
  Using cached ruamel.yaml-0.17.16-py3-none-any.whl (109 kB)
Collecting intel-openmp
  Using cached intel_openmp-2021.3.0-py2.py3-none-win_amd64.whl (3.3 MB)
Collecting six~=1.11.0
  Using cached six-1.11.0-py2.py3-none-any.whl (10 kB)
Requirement already satisfied: certifi in c:\anaconda\anaconda3\envs\simba\lib\site-packages (from deeplabcut==2.0.9->simba-uw-tf) (2021.5.30)
Collecting ipython-genutils~=0.2.0
  Using cached ipython_genutils-0.2.0-py2.py3-none-any.whl (26 kB)
Collecting wheel~=0.31.1
  Using cached wheel-0.31.1-py2.py3-none-any.whl (41 kB)
Collecting python-dateutil~=2.7.3
  Using cached python_dateutil-2.7.5-py2.py3-none-any.whl (225 kB)
Requirement already satisfied: setuptools in c:\anaconda\anaconda3\envs\simba\lib\site-packages (from deeplabcut==2.0.9->simba-uw-tf) (52.0.0.post20210125)
Collecting ipython~=6.0.0
  Using cached ipython-6.0.0-py3-none-any.whl (736 kB)
Collecting numpy==1.18.1
  Using cached numpy-1.18.1-cp36-cp36m-win_amd64.whl (12.8 MB)
INFO: pip is looking at multiple versions of simba-uw-tf to determine which version is compatible with other requirements. This could take a while.
INFO: This is taking longer than usual. You might need to provide the dependency resolver with stricter constraints to reduce runtime. If you want to abort this run, you can press Ctrl + C to do so. To improve how pip performs, tell us what happened here: https://pip.pypa.io/surveys/backtracking
Collecting scikit-learn~=0.19.2
  Using cached scikit_learn-0.19.2-cp36-cp36m-win_amd64.whl (4.4 MB)
Collecting easydict~=1.7
  Using cached easydict-1.9.tar.gz (6.4 kB)
Collecting numpy~=1.14.5
  Using cached numpy-1.14.6-cp36-none-win_amd64.whl (13.4 MB)
Collecting tensorpack~=0.9.7.1
  Using cached tensorpack-0.9.7.1-py2.py3-none-any.whl (286 kB)
Collecting chardet~=3.0.4
  Using cached chardet-3.0.4-py2.py3-none-any.whl (133 kB)
Collecting click
  Using cached click-8.0.1-py3-none-any.whl (97 kB)
Collecting colour
  Using cached colour-0.1.5-py2.py3-none-any.whl (23 kB)
Collecting jinja2
  Using cached Jinja2-3.0.1-py3-none-any.whl (133 kB)
Collecting attrs>16.0.0
  Using cached attrs-21.2.0-py2.py3-none-any.whl (53 kB)
Collecting imbalanced-learn
  Using cached imbalanced_learn-0.8.0-py3-none-any.whl (206 kB)
INFO: pip is looking at multiple versions of imblearn to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of h5py to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of graphviz to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of eli5 to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of dtreeviz to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of deepposekit to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of deeplabcut to determine which version is compatible with other requirements. This could take a while.
ERROR: Cannot install simba-uw-tf and simba-uw-tf==1.0.3 because these package versions have conflicting dependencies.

The conflict is caused by:
    simba-uw-tf 1.0.3 depends on scikit-learn==0.22.2
    deeplabcut 2.0.9 depends on scikit-learn~=0.19.2

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/user_guide/#fixing-conflicting-dependencies
```

### After **pip install simba-uw-tf**:

```
(Simba) C:\Users\************>pip install simba-uw-tf==1.3.11
Collecting simba-uw-tf==1.3.11
  Using cached Simba_UW_tf-1.3.11-py3-none-any.whl (7.3 MB)
Collecting seaborn==0.9.0
  Using cached seaborn-0.9.0-py3-none-any.whl (208 kB)
Collecting dash-color-picker==0.0.1
  Using cached dash_color_picker-0.0.1-py3-none-any.whl
Collecting numexpr==2.6.9
  Using cached numexpr-2.6.9-cp36-none-win_amd64.whl (91 kB)
Collecting pyarrow==0.17.1
  Using cached pyarrow-0.17.1-cp36-cp36m-win_amd64.whl (21.6 MB)
Collecting dash-colorscales==0.0.4
  Using cached dash_colorscales-0.0.4-py3-none-any.whl
Collecting imgaug==0.4.0
  Using cached imgaug-0.4.0-py2.py3-none-any.whl (948 kB)
Collecting plotly==4.9.0
  Using cached plotly-4.9.0-py2.py3-none-any.whl (12.9 MB)
Collecting tabulate==0.8.3
  Using cached tabulate-0.8.3-py3-none-any.whl
Collecting opencv-python==3.4.5.20
  Using cached opencv_python-3.4.5.20-cp36-cp36m-win_amd64.whl (38.3 MB)
Collecting cefpython3==66.0
  Using cached cefpython3-66.0-py2.py3-none-win_amd64.whl (68.2 MB)
Collecting pandas==0.25.3
  Using cached pandas-0.25.3-cp36-cp36m-win_amd64.whl (9.0 MB)
Collecting deeplabcut==2.0.9
  Using cached deeplabcut-2.0.9-py3-none-any.whl (187 kB)
Collecting h5py==2.9.0
  Using cached h5py-2.9.0-cp36-cp36m-win_amd64.whl (2.4 MB)
Collecting scipy==1.1.0
  Using cached scipy-1.1.0-cp36-none-win_amd64.whl (31.1 MB)
Collecting pyyaml==5.3.1
  Using cached PyYAML-5.3.1-cp36-cp36m-win_amd64.whl (215 kB)
Collecting scikit-image==0.14.2
  Using cached scikit_image-0.14.2-cp36-none-win_amd64.whl (24.6 MB)
Collecting xgboost==0.90
  Using cached xgboost-0.90-py2.py3-none-win_amd64.whl (18.3 MB)
Collecting graphviz==0.11
  Using cached graphviz-0.11-py2.py3-none-any.whl (17 kB)
Collecting numba==0.48.0
  Using cached numba-0.48.0-cp36-cp36m-win_amd64.whl (2.1 MB)
Collecting dash==1.14.0
  Using cached dash-1.14.0-py3-none-any.whl
Collecting shapely==1.7
  Using cached Shapely-1.7.0-cp36-cp36m-win_amd64.whl (1.0 MB)
Collecting dash-core-components==1.10.2
  Using cached dash_core_components-1.10.2-py3-none-any.whl
Collecting imutils==0.5.2
  Using cached imutils-0.5.2-py3-none-any.whl
Collecting wxpython==4.0.4
  Using cached wxPython-4.0.4-cp36-cp36m-win_amd64.whl (23.0 MB)
Collecting eli5==0.10.1
  Using cached eli5-0.10.1-py2.py3-none-any.whl (105 kB)
Collecting Pillow==5.4.1
  Using cached Pillow-5.4.1-cp36-cp36m-win_amd64.whl (1.9 MB)
Collecting dash-html-components==1.0.3
  Using cached dash_html_components-1.0.3-py3-none-any.whl
Collecting dtreeviz==0.8.1
  Using cached dtreeviz-0.8.1-py3-none-any.whl
Collecting imblearn==0.0
  Using cached imblearn-0.0-py2.py3-none-any.whl (1.9 kB)
Collecting matplotlib==3.0.3
  Using cached matplotlib-3.0.3-cp36-cp36m-win_amd64.whl (9.1 MB)
Collecting scikit-learn==0.22.2
  Using cached scikit_learn-0.22.2-cp36-cp36m-win_amd64.whl (6.5 MB)
Collecting yellowbrick==0.9.1
  Using cached yellowbrick-0.9.1-py2.py3-none-any.whl (282 kB)
Collecting tqdm==4.30.0
  Using cached tqdm-4.30.0-py2.py3-none-any.whl (47 kB)
Collecting shap==0.35.0
  Using cached shap-0.35.0-cp36-cp36m-win_amd64.whl (309 kB)
Collecting deepposekit==0.3.5
  Using cached deepposekit-0.3.5-py3-none-any.whl
Collecting statsmodels==0.9.0
  Using cached statsmodels-0.9.0-cp36-cp36m-win_amd64.whl (7.0 MB)
Collecting dash-renderer==1.6.0
  Using cached dash_renderer-1.6.0-py3-none-any.whl
Collecting Flask>=1.0.2
  Using cached Flask-2.0.1-py3-none-any.whl (94 kB)
Collecting future
  Using cached future-0.18.2-py3-none-any.whl
Collecting flask-compress
  Using cached Flask_Compress-1.10.1-py3-none-any.whl (7.9 kB)
Collecting dash-table==4.9.0
  Using cached dash_table-4.9.0-py3-none-any.whl
Collecting easydict~=1.7
  Using cached easydict-1.9.tar.gz (6.4 kB)
INFO: pip is looking at multiple versions of dash-table to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of dash-renderer to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of dash-html-components to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of dash-core-components to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of dash-colorscales to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of dash-color-picker to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of <Python from Requires-Python> to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of dash to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of cefpython3 to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of simba-uw-tf to determine which version is compatible with other requirements. This could take a while.
ERROR: Cannot install simba-uw-tf and simba-uw-tf==1.3.11 because these package versions have conflicting dependencies.

The conflict is caused by:
    simba-uw-tf 1.3.11 depends on scikit-learn==0.22.2
    deeplabcut 2.0.9 depends on scikit-learn~=0.19.2

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/user_guide/#fixing-conflicting-dependencies
```

**Comment by quirinkrabi** [comment]
Hi!

I got it to work with "pip install simba-uw-tf-dev".

Sorry for bothering!

Best,
Quirin

**Comment by sgoldenlab** [comment]
Excellent, thanks Quirin - was just about to add that the solution is in FAQ entry 11: https://github.com/sgoldenlab/simba/blob/master/docs/FAQ.md#11-when-i-install-or-update-simba-i-see-a-bunch-or-messages-in-the-console-telling-there-me-about-some-dependency-conflicts-the-messages-may-look-a-little-like-this

But yes, we strongly recommend the dev version over the TF version.
